enclave_manager/avalon_enclave_manager/wpe/wpe_requester.py

```diff
@@ -200,11 +201,11 @@ def preprocess_work_order(self, wo_request, encryption_key):
             if self._verify_res_signature(wo_response_json,
                                           self._worker.verification_key,
                                           wo_req["params"]["requesterNonce"]):
-                decrypted_res = worker_encrypt.decrypted_response(
-                    wo_response_json, session_key, session_iv)
+                decrypted_res = worker_encrypt.decrypt_work_order_data_json(
```

Comment:
These 2 statements could have been commoned out 204-208. This shows up when we have to make the same change at 3 places. Maintainability.

Reply:
Agree. I will write a common method for this.